torchvision/models/resnet.py

```diff
@@ -294,3 +296,15 @@ def resnext101_32x8d(pretrained=False, progress=True, **kwargs):
     kwargs['width_per_group'] = 8
     return _resnet('resnext101_32x8d', Bottleneck, [3, 4, 23, 3],
                    pretrained, progress, **kwargs)
+
+
+def wide_resnet50_2(pretrained=False, progress=True, **kwargs):
+    kwargs['width_per_group'] = 64 * 2
```

Comment:
Can you add some documentation for those functions? They can be very similar to the ones from ResNet